server/conn.go

```diff
@@ -822,7 +824,8 @@ func (cc *clientConn) handleLoadData(ctx context.Context, loadDataInfo *executor
 		}
 		return errors.Trace(err)
 	}
-	return errors.Trace(txn.Commit(sessionctx.SetCommitCtx(ctx, loadDataInfo.Ctx)))
+
+	return errors.Trace(loadDataInfo.Ctx.RefreshTxnCtx(ctx))
```

Comment:
What the difference does this change make?

Reply:
RefreshTxnCtx will start a new transaction. We should not start a new transaction here.